lte/gateway/python/magma/pipelined/app/enforcement_stats.py

```diff
@@ -576,6 +576,8 @@ def _get_usage_from_flow_stat(self, flow_stats):
             ipv6_addr = _get_ipv6(flow_stat)
 
             local_f_teid_ng = _get_ng_local_f_id(flow_stat)
+            if local_f_teid_ng and report_flag:
```

Comment:
this looks too hacky to me. I see this IGNORE_STATS facility used on line 570, specifying this in the policy itself makes more sense. @koolzz @pshelar what do you think?

Reply:
second best option is to remove report_flag and put a comment over the check `if local_f_teid_nf` stating that when stat reporting feature is added for 5G, this check should be removed.

Reply:
ideally, we should have used NO_TRACKING for this, but I see that in implementation it is not used inline with its meaning.

Reply:
Yes, it's a temp fix. We will add stat reporting feature for 5G on coming week then will remove the check.  The _get_usage_from_flow_stat is also used for delete_old_flows. So if report_flag is false means adding 5G flows as well for deleting old stats flows.

Reply:
Ignore stats is internal implementation to allow few packets prior to DPI classification, but overall I'm not sure whats happening in this pr

Reply:
Just leave if local_teid and completely remove the  report flag and I'll accept